rts/Sim/Units/Scripts/CobThread.h

```diff
@@ -107,31 +107,32 @@ class CCobThread
 
 	void LuaCall();
 
-	bool PushCallStack(CallInfo v) { return (callStackSize < callStack.size() && PushCallStackRaw(v)); }
-	bool PushDataStack(     int v) { return (dataStackSize < dataStack.size() && PushDataStackRaw(v)); }
+	void PushCallStack(CallInfo v) {
+		assert(callStack.size() < 64);
```

Comment:
Do these arbitrary limits still make sense given it's no longer a static array? There's probably some value in preventing infinite recursion. Could use a comment at any rate
```suggestion
		assert(callStack.size() < 64); // arbitrary historical limit, prevents infinite recursion and does XYZ
```

Reply:
Yeah, and heh, I've noticed that for benchmarking I've compiled without asserts enabled. In debug mode BAR is actually hitting those limits. Will prepare some dump with more data.

Reply:
64-deep calls sounds like it could be an error in the script (unless I'm misinterpreting it)

Reply:
I've dropped those asserts.